nvflare/private/fed/server/fed_server.py

```diff
@@ -665,6 +671,11 @@ def quit_client(self, request: Message) -> Message:
             headers = {CellMessageHeaderKeys.MESSAGE: "Removed client"}
             return self._generate_reply(headers=headers, payload=None, fl_ctx=fl_ctx)
 
+    def process_error_log(self, request: Message):
+        payload = request.payload
+        client = request.get_header(key=MessageHeaderKey.ORIGIN)
+        print(f"Error log from {client}: {payload}")
```

Comment:
Is this for debugging purpose?

Reply:
Yes, this is going to be changed in the next PR for handling this on the server.